tests/phpunit/tests/html-api/wpHtmlProcessorSemanticRules.php

```diff
@@ -16,6 +16,110 @@ class Tests_HtmlApi_WpHtmlProcessorSemanticRules extends WP_UnitTestCase {
 	 * RULES FOR "IN BODY" MODE
 	 *******************************************************************/
 
+	/**
+	 * Verifies that when encountering an end tag for which there is no corresponding
+	 * element in scope, that it skips the tag entirely.
+	 *
+	 * @ticket {TICKET_NUMBER}
```

Comment:
```suggestion
	 * @ticket 58961
```

Reply:
thanks for noticing this. added it in 5ad4fd616e